druid/ingestion-spec/rollups/ml_project_index_rollup.json

```diff
@@ -0,0 +1,350 @@
+{
+    "type": "kafka",
+    "spec": {
+      "dataSchema": {
+        "dataSource": "ml-project-rollup",
+        "timestampSpec": {
+          "column": "project_updated_date",
+          "format": "iso",
+          "missingValue": null
+        },
+        "dimensionsSpec": {
+          "dimensions": [
+            {
+              "type": "string",
+              "name": "project_title",
+              "multiValueHandling": "SORTED_ARRAY",
+              "createBitmapIndex": true
+            },
+            {
+              "type": "string",
+              "name": "area_of_improvement",
```

Comment:
If the field is an array as defined in the ioConfig, only the field name needs to be included here. The SORTED_ARRAY for every field is not necessary. If it is a normal string or double field, include the field name and specify the data type.

Reply:
This is fixed now. Please check